gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/HotReloadTest.kt

```diff
@@ -6,18 +6,21 @@ import org.jetbrains.compose.ComposeBuildConfig
 import org.jetbrains.compose.desktop.application.internal.ComposeProperties
 import org.jetbrains.compose.test.utils.GradlePluginTestBase
 import org.jetbrains.compose.test.utils.checks
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.fail
 import org.junit.jupiter.api.Test
 import kotlin.concurrent.thread
 
 class HotReloadTest : GradlePluginTestBase() {
+    @Disabled
```

Comment:
Please add some comment/TODO with explanation / link to issue

Reply:
Added